packages/manager/modules/iam/src/components/createPolicy/createPolicy.controller.js

```diff
@@ -211,6 +211,11 @@ export default class CreatePolicyController {
         (resourceType, index, list) =>
           Boolean(resourceType) && list.indexOf(resourceType) === index,
       );
+      this.model.permissionsGroups = this.policy.permissionsGroups.map(
+        ({ urn }) => {
+          return { urn };
+        },
+      );
```

Comment:
```suggestion
      this.model.permissionsGroups = this.policy.permissionsGroups.map(
        ({ urn }) => urn,
      );
```
Can be simplified a bit

Reply:
Done